src/config/jest.js

```diff
@@ -7,8 +7,8 @@ module.exports = {
     '**/test/**/*.js'
   ],
   testPathIgnorePatterns: [
-    '/node_modules/',
-    '/src/'
+    '<rootDir>/node_modules/',
```

Comment:
we should keep just `node_modules`, as otherwise things like the examples in js-ipfs will be scanned

Reply:
@dignifiedquire true, didn't think about that. Thanks!